packages/security/src/components/Wizard/Wizard-story.js

```diff
@@ -238,6 +240,27 @@ const markdown = (useDefault = true) =>
   `;
 
 storiesOf(patterns('Wizard#legacy'), module)
+  .addDecorator((Story) => (
+    <>
+      <InlineNotification
```

Comment:
Another one where the InlineNotification is hidden by other elements in the story.